Backend/src/routes/api.js

```diff
@@ -17,24 +17,29 @@ export default router;
 //--------------TCP-------------------------
 
 import { Socket } from "net";
-// const { Buffer } = require("buffer");
-const car_port = 4003;
+const car_port = 4003; // Port for TCP connection
```

Comment:
Constant variables should be all caps


```suggestion
const CAR_PORT = 4003; // Port for TCP connection
```

Honestly they should be kept in some one singular file but let's discuss later.

Reply:
I changed constants to all caps, and I made a note/task to add a constants file.